scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/server/RestClient.java

```diff
@@ -785,4 +784,10 @@ void putThirdPartyCredential(@HeaderParam("sessionid") String sessionId, @PathPa
     @Produces(MediaType.APPLICATION_JSON)
     Map<Object, Object> getSchedulerPortalDisplayProperties(@HeaderParam("sessionid") String sessionId);
 
+    @GET
+    @Path("job/{jobid}/permission/{method}")
+    @Produces(MediaType.APPLICATION_JSON)
+    InputStream jobVariablesAccess(@HeaderParam("sessionid") String sessionId, @PathParam("jobid") String jobId,
```

Comment:
please rename this to checkJobPermissionMethod

Reply:
done